egs/tedlium/s5/local/chain/run_tdnn.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+#
+# This script requires that you have run the toplevel run.sh script in TEDLIUM up to stage 7.
+#
+# Results: (Run for x in exp/chain/tdnn/decode*; do [ -d $x ] && grep Sum $x/score_*/*.sys | utils/best_wer.sh; done 2>/dev/null)
+## %WER 14.1 | 507 17792 | 88.6 7.3 4.1 2.7 14.1 92.9 | 0.075 | exp/chain/tdnn/decode_dev/score_10_0.5/ctm.filt.filt.sys
+## %WER 13.3 | 507 17792 | 89.7 6.9 3.4 2.9 13.3 92.1 | 0.000 | exp/chain/tdnn/decode_dev_rescore/score_10_0.0/ctm.filt.filt.sys
+## %WER 13.8 | 1155 27512 | 89.4 7.5 3.1 3.2 13.8 87.9 | 0.101 | exp/chain/tdnn/decode_test/score_10_0.0/ctm.filt.filt.sys
+## %WER 12.9 | 1155 27512 | 90.1 6.6 3.3 2.9 12.9 86.1 | 0.043 | exp/chain/tdnn/decode_test_rescore/score_10_0.0/ctm.filt.filt.sys
+# The final WER (rescored WER on the test set) is what we are interested in.
+
+# To reproduce the setup used in the paper, set the following variables:
+# affix=_more_ce
+# relu_dim=525
+# xent_regularize=0.2
```

Comment:
What do you mean by "portion"? Do you mean you would like to see the number of parameters of each of the two models whose results I provide?